src/content/concepts/module-federation.md

```diff
@@ -144,12 +144,14 @@ Many applications share a common components library which could be built as a co
 
 The container interface supports `get` and `init` methods.
 `init` is a `async` compatible method that is called with one argument: the shared scope object. This object is used as a shared scope in the remote container and is filled with the provided modules from a host.
-It can be leveraged to connect remote containers to a host container dynamically at runtime
+It can be leveraged to connect remote containers to a host container dynamically at runtime.
+
+__init.js__
 
 ```js
-(async () =>{
-  // Initializes the share scope. This fills it with known provided modules from this build and all remotes
-  await __webpack_initialize_sharing__('default');
+(async () => {
+  // Initializes the shared scope. Fills it with known provided modules from this build and all remotes
+  await __webpack_init_sharing__('default');
```

Comment:
incorrect naming. My use is accurate. Its called init, not initialize. Original comments had a typo, this is the actual function name as of the last time i used this (beta 22)

Reply:
yeah thats why i am changing it from `__webpack_initialize_sharing__` to `__webpack_init_sharing__`